integration-cli/docker_cli_daemon_test.go

```diff
@@ -2540,14 +2540,14 @@ func (s *DockerDaemonSuite) TestExecWithUserAfterLiveRestore(c *testing.T) {
 
 func (s *DockerDaemonSuite) TestRemoveContainerAfterLiveRestore(c *testing.T) {
 	testRequires(c, DaemonIsLinux, overlayFSSupported, testEnv.IsLocalDaemon)
```

Comment:
Is the `overlayFSSupported` still need if we remove the `--storage-driver` option?

Reply:
Removed ;)